packages/cli/src/lib/aws-utils.ts

```diff
@@ -2,12 +2,20 @@ import AWS from 'aws-sdk'
 import {AddPermissionRequest} from 'aws-sdk/clients/lambda'
 
 
-export async function addPermission(arn: string, principal = '906037444270'): Promise<string> {
+export async function addPermission(arn: string, principal = '906037444270', statementId = 'smartthings'): Promise<string> {
 	const segs = arn.split(':')
 	if (segs.length < 7) {
 		return 'Invalid Lambda ARN'
 	}
 
+	if (!principal) {
+		principal = '906037444270'
+	}
+
+	if (!statementId) {
+		statementId = 'smartthings'
+	}
```

Comment:
You actually don't need this because you have the default values. (Even passing a literal `undefined` as one of the values will use its default.)

Reply:
Ha! I was thinking it would behave like a null, but not setting a flag does return undefined, so I removed these for clarity